SPRINT-STRUCTURE.md

```diff
@@ -1,115 +1,73 @@
-- Day 4
-  - [Continuous Deployment](continuous-deployment/readme.md) (extend, much more explanation needed)
-    - Conceptual, Purpose, Goals, Impact and Measures
-      - Which environments (ci, test, stage)? For each environment...
-        - Conceptually whats is the purpose for each env
-        - When should deployment trigger
-        - Pre-deployment approvers
-        - Sign off for promotion
-    - Recipes for Setting up CD Pipelines
-      - Azure DevOps
-  - Asserting Test Cases and Automation
-- Day 5
-  - Sprint Demo
-  - [Retrospectives](retrospectives/readme.md)
-    - Conceptual
-      - Inputs (Requirements to have ready before meeting)
-      - Participants required
-      - Outputs (Decisions, actions to conclude meeting)
-    - Guide for retrospective facilitator
-      - Timeline for 1 hour retro
-      - Tips for sticking to time
-      - Voting for action items
-    - Recipes
-      - Remote retros using ADO Retrospectives
-      - Remote retros using Retrium
-  - [Grooming](backlog-management/grooming/readme.md)
-    - Conceptual
-      - Inputs (Requirements to have ready before meeting)
-      - Participants required
-      - Outputs (Decisions, actions to conclude meeting)
-    - Definition of ready for stories
-      - Examples of well defined acceptance criteria
-      - Can the story be tested as written
-      - Can it be completed within a sprint
-      - Is it dependent on other stories
-    - [Estimation](sprint-planning/estimation/readme.md)
-      - Resolving estimation conflicts (two people are sizing differently))
+### Day 1
 
-## Assumptions
+- [ ] [Plan the first sprint](sprint-planning/readme.md)
+  - Agree on a sprint goal, and how to measure the sprint progress
+  - Determine team capacity
+  - Assign user stories to the sprint and split user stories into tasks
+  - Set up Work in Progress (WIP) limits
+- [ ] [Decide on test frameworks and discuss test strategies](test-first-development/readme.md)
+  - Discuss the purpose and goals of tests and how to measure test coverage
+  - Agree on how to separate unit tests from integration, load and smoke tests
+- [ ] [Decide on branch naming for feature branches](source-control/feature-branching/readme.md)
+- [ ] [Discuss security needs and verify that secrets are kept out of source control](continuous-deployment/secrets-management/recipes/azure-devops/secrets-per-branch.md)
 
-This workflow makes the following assumptions about the development environment
+### Day 2
 
-- Team is using git for version control.
+- [ ] [Set up Source Control](source-control/readme.md)
+  - Agree on [best practices for commits](https://github.com/microsoft/code-with-engineering-playbook/tree/master/source-control#commit-best-practices)
+- [ ] [Set up basic Continuous Integration with linters and automated tests](continuous-integration/readme.md)
+- [ ] [Set up meetings for Daily Standups and decide on a scrum master](stand-ups/readme.md)
```

Comment:
Should we add the other scrum ceremonies as well? Planning Review, Retro? Or just generalize it with Setup meetings and decide on scrum master?

Reply:
They are already in there